### ISSUE TITLE
Return value of barrelstrength\sproutbasereports\services\Reports::getReport() must be an instance of barrelstrength\sproutbasereports\elements\Report, null returned

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-reports#64]
Running into issue when visiting admin/dashboard

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_y8lf)

**Comment by harrymee123** [comment]
Hi,

Is there a fix for this. I'm getting the same thing.

Cheers

**Comment by BenParizek** [comment]
Hi @harrymee123. The other user that reported this mentioned they think it might have done something to do with Craft itself, and we didn't get any follow-ups after that.

While I may need more info to troubleshoot, a good place to start may be to remove your `vendor/` folder and `composer.lock` file and re-run `composer install` to make sure you have all the latest packages and dependencies.

To troubleshoot further, can you answer the following questions:

- What version of Sprout Reports are you running? 
- Did you recently update anything (via command line or CP)?
- Do you have any Sprout Reports dashboard widgets?
- Could you share your composer.json or db (or even just the Sprout Reports tables in the db if you can't share the whole thing) so we can get a better idea of how to recreate the issue? (you can send this to sprout@barrelstrengthdesign.com)

**Comment by harrymee123** [comment]
Hi @BenParizek. Super sorry on the late reply, really busy on another project. 

I tried, removing the `vendor/` folder and `composer.lock` file and running `composer install` still have the issue.

- I'm running Sprout Reports 1.2.8
- I don't remember doing anything with the cli or CP but I may have ran a composer update.
- I don't but I did add one and then remove it as it didn't seem to show any data. It was then the next time I visited the dashboard the error showed up.
- I've exported the database tables and sent them over to the email you supplied.

I had a quick glance at the `sproutreports_reports` table and noticed a report that I deleted was still there. The one that id is 14. That was the report that I added as a widget before, deleting the widget and the report.

Cheers,
